pimd/pim_str.h

```diff
@@ -37,6 +40,26 @@
 #define pim_inet4_dump prefix_mcast_inet4_dump
 #define pim_str_sg_set prefix_sg2str
 
+static inline void pim_addr_copy(pim_addr *dest, pim_addr *source)
+{
+	dest->s_addr = source->s_addr;
+}
+
+static inline int pim_is_addr_any(pim_addr addr)
+{
+	return (addr.s_addr == INADDR_ANY);
+}
+
+static inline int pim_addr_compare(pim_addr addr1, pim_addr addr2)
```

Comment:
Nit: `_compare` => `_cmp` for 3 reasons:

- with `_cmp` it is signaled that the return value is like `memcmp` and `strcmp`, i.e. -/0/+.
- it's shorter
- existing uses in FRR of `_cmp` outnumber `_compare`

```
frr $ git grep _compare '*.h' | wc -l
88
frr $ git grep _cmp '*.h' | wc -l 
141
```

Reply:
Fixed.